plugins/image-source/image-source.c

```diff
@@ -214,12 +214,14 @@ static void image_source_tick(void *data, float seconds)
 
 
 static const char *image_filter =
-	"All formats (*.bmp *.tga *.png *.jpeg *.jpg *.gif);;"
+	"All formats (*.bmp *.tga *.png *.jpeg *.jpg *.gif *.psd);;"
 	"BMP Files (*.bmp);;"
 	"Targa Files (*.tga);;"
 	"PNG Files (*.png);;"
 	"JPEG Files (*.jpeg *.jpg);;"
-	"GIF Files (*.gif)";
+	"GIF Files (*.gif);;"
+	"Photoshop Files (*.psd);;"
```

Comment:
Not sure if it makes more sense to say "Photoshop Files" or "PSD Files".  The only item in the list that doesn't follow the pattern of "\<Extension> Files" is Targa.  As an aside or historical note, TARGA is apparently an acronym.

Reply:
ok changed photoshop to psd